pymc/distributions/__init__.py

```diff
@@ -116,8 +117,8 @@
     "Uniform",
     "Flat",
     "HalfFlat",
-    "TruncatedNormal",
```

Comment:
Small nitpick: your vscode also changed this without need

Reply:
Ha ha no, _I_ did that because I like alphabetical order 🙈